identity/client/src/pages/roles/modals/DeleteModal.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * @license Identity
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under one or more contributor license
+ * agreements. Licensed under a proprietary license. See the License.txt file for more information. You may not use this
+ * file except in compliance with the proprietary license.
+ */
+
```

Comment:
The license header is from old Identity. We can delete if for now or include the new license header.